### DISCUSSION TITLE
Use models from other RE games

### DISCUSSION THREAD

**Discussion by Hynisel** [IntelOrca/biorand#438] [General]
Hello, would it be possible to use a model from RE 2 (Hunk) in RE3 or 1 ?

Thank you.

**Comment by IntelOrca** [comment]
Yes, but it requires a lot of work to convert between the games. Particularly RE 1 which is significantly different. I am working a tool to try to automate the process as much as possible.

**Reply by Hynisel** [reply]
Thank you for your reply, I think many people would love to play their favourite character through the games, I believe it goes well with the randomizer. I will patiently wait for this possibility.
Thank you for your work!

**Comment by IntelOrca** [comment]
@Hynisel Hunk is now playable in RE 3

**Reply by Hynisel** [reply]
@IntelOrca
Thank you for updating me about that. I was testing it just before and it presents some glitches. I will create a bug report in the appropriate section with the details as soon as I can.
Thank you again.
